grammar.js

```diff
@@ -263,7 +266,13 @@ module.exports = grammar({
       field('name', choice($.constant, $.scope_resolution)),
       field('superclass', optional($.superclass)),
       $._terminator,
-      $._body_statement
+      optional(field("namespace_body", $.body)),
+      'end'
+    ),
+
+    body: $ => choice(
```

Comment:
I think we should name this rule `body_statement` to match up with the terminology used in [parse.y](https://github.com/ruby/ruby/blob/master/parse.y#L1416)